### ISSUE TITLE
Wrong expansion depending on order of attributes

### ISSUE THREAD

**Issue by ghost** [ambethia/emmet-jsx-css-modules#4]
When using the module for trivial expansions, it works correctly.
E.g: `p#foo.bar` expands to

``` html
<p id="foo" className={style.bar}></p>
```

When used for not-so-trivial expansions, the property closing breaks.
Examples:
`p.bar#foo` expands to

``` html
<p className={style.bar" id="foo}></p>
```

`p#foo.bar[name=bas]` expands to

``` html
<p id="foo" className={style.bar" name="bas}></p>
```

**Comment by ambethia** [comment]
Hmm. Do you have an idea for a fix/PR?

**Comment by ghost** [comment]
Truth be told, I haven't looked into the code. Though if it's being done through Regexp, maybe having a non greedy expression would help. I'll take a look into it when possible.

**Comment by ambethia** [comment]
That was exactly it. Fixed in the newest release (63a7ea5648a1da9645bcfd74f085c0c2f13c8dd8)

Thanks!
